packages/snjs/lib/services/api/http_service.ts

```diff
@@ -98,6 +122,19 @@ export class SNHttpService extends PureService {
     return request;
   }
 
+  private getCorrectHeaderForEnvironment(environment: Environment): string {
```

Comment:
```suggestion
  private headerForEnvironment(environment: Environment): string {
```

Reply:
```suggestion
  private versionHeaderName(): string {
```

This function also doesn't need to take a param since this.environment is already available to it.

Reply:
I'm usually prepending `get` to such method names so that they differ from property names. Updated. :)